TACC/Frontera-Portal
# task/FP-734 move accounts route under workbench

## Overview: ##
Moving accounts under workbench (i.e.  `workbench/account`)

See ticket for more info but this is to help us wall off parts of the portal for the on-boarding process.

## PR Status: ##


## Related Jira tickets: ##

* [FP-734](https://jira.tacc.utexas.edu/browse/FP-734)

## Summary of Changes: ##
Moved route from `/accounts/profile` to `workbench/account`
## Testing Steps: ##
1. From dashboard click on "manage account"
2. Ensure that it looks exactly like it did before
3. Ensure that it behaves as it did before
4. Ensure the route is  `workbench/account`
